tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -149,6 +150,13 @@ def keyboard(self):
         from gaiatest.apps.keyboard.app import Keyboard
         return Keyboard(self.marionette)
 
+    @property
+    def manifest_url(self):
+        return '{}{}.{}/manifest.webapp'.format(self.DEFAULT_PROTOCOL, format(self.__class__).lower(), self.DEFAULT_APP_HOSTNAME)
```

Comment:
DEFAULT_PROTOCOL is only mentioned once. I don't think it already exists, does it?